### DISCUSSION TITLE
Automatic replica creation for databases

### DISCUSSION THREAD

**Discussion by Tilo-K** [coollabsio/coolify#4278] [Feature Requests]
I would like to have an easy way to create a replica instance of my database (in my case postgres, but would also be cool for the others) on another server. So that I can use for example the "Resource Operations" to not only clone the database but also set up a replica instance.

**Comment by joaopcm** [comment]
This would be great

**Comment by enezatech** [comment]
This would be awesome

**Comment by majcek210** [comment]
this would be rlly good for production uses (:

**Comment by drpark** [comment]
amazing
